tutorials/incorporating_oli_calculations.ipynb

```diff
@@ -487,9 +484,9 @@
  "metadata": {
   "celltoolbar": "Slideshow",
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "watertap-dev",
    "language": "python",
-   "name": "python3"
+   "name": "watertap-dev"
```

Comment:
@veccp I think this is causing failures in the oli-api-ci workflows. Sorry about the confusion. I'll try to open a PR next week so that the CI jobs for notebooks are consistent with the `watertap-dev` name we have in the documentation (and will cause an error otherwise).

Reply:
This would be #1320

Reply:
@lbianchi-lbl Sounds good, are you saying the notebook kernel should be `watertap-dev` and not `Python 3 (ipykernel)`?